### PR TITLE
Remove pool

### PR DESCRIPTION
**Pull request by chohannesthurn**

### CODE REVIEW

**Comment by jstarek** [comment]
Hi, first of all, I have to admit that when I read the title of your pull request, "remove pool", I hesitated a bit before giving it the go-ahead for testing.  Please use titles that sound less like "hey, I'm going to break dCache, and here's how" in the future ;-)

Secondly, I'm going to trigger testing now, but before accepting the pull request, I'm going to have to ask you to change it and provide the missing documentation. Thanks!

**Comment by jstarek** [comment]
ok to test

File: modules/dcache/src/main/java/diskCacheV111/poolManager/PoolSelectionUnitV2.java [hunk 1]
```diff
@@ -2749,8 +2750,16 @@ public String ac_psu_ls_netunits(Args args)
     @AffectsSetup
     public String ac_psu_remove_pool_$_1(Args args)
     {
-        removePool(args.argv(0));
-        return "";
+	if(Glob.isGlob(args.argv(0)))
+	{
+		Glob glob = new Glob(args.argv(0));	
+		List<String> names = getPools(glob.toPattern()).stream().map((Pool pool) -> pool.getName()).collect(Collectors.toList());
+		for(String name : names)
```
Comment (kofemann) [inline modules/dcache/src/main/java/diskCacheV111/poolManager/PoolSelectionUnitV2.java]: please use curly bases  even with e single statement, e.g:

for (...) {
  removePool(name);
}

File: modules/dcache/src/main/java/diskCacheV111/poolManager/PoolSelectionUnitV2.java [hunk 2]
```diff
@@ -2749,8 +2750,16 @@ public String ac_psu_ls_netunits(Args args)
...
+			removePool(name);
+	}
+	else
```
Comment (kofemann) [inline modules/dcache/src/main/java/diskCacheV111/poolManager/PoolSelectionUnitV2.java]: same here

**Comment by kofemann** [comment]
thanks!

**Pull request merged**